cli/session.go

```diff
@@ -287,3 +296,30 @@ func getMetrics(toComplete string) []string {
 
 	return metrics
 }
+
+func getCloudServices(_ string) []string {
```

Comment:
```suggestion
func getCloudServices() []string {
```

Reply:
This is intentional (future proof). The `toComplete` holds the typed input, we might want to include this in the request as a filter.